src/cron.ts

```diff
@@ -183,18 +208,29 @@ export const cron = async (startDate: Date, owner: string, repo: string, octokit
       }
 
       // if we don't find a match, we search for a closed referenda
-      const completedMatch = completedReferendas.find(
+      const completedMatch = finishedReferendas.find(
         ({ hash }) => hash === tx.method.hash.toHex() || hash === tx.method.toHex(),
       );
       if (completedMatch) {
-        logger.info(`Found completed referenda for PR #${pr}`);
+        logger.info(`Found finished referenda for PR #${pr} with state ${completedMatch.state}`);
         const command = `/rfc process ${completedMatch.executedHash}`;
-        const msg = "PR can be merged. Write the following command to trigger the bot" + `\n\n\`${command}\``;
+        const timeOutMsg = "Referenda voting has timed out";
+        const finishedMsg =
+          (completedMatch.state === "Executed"
+            ? "PR can be merged."
+            : `Referenda state is \`${completedMatch.state}\`. PR can be closed.`) +
```

Comment:
```suggestion
            : `Referenda state is \`${completedMatch.state}\`.`) +
```

So in this case they can decide to run another referendum, closing PR is not the only option.
And the `/rfc process` command should not be proposed here because it will not do anything.